packages/nimble-tokens/source/styledictionary/config.js

```diff
@@ -1,49 +1,49 @@
 const StyleDictionary = require('style-dictionary');
```

Comment:
Most of the changes in this file are from running `npm run format`. The other changes were:
1. suppress [no-control-regex](https://eslint.org/docs/latest/rules/no-control-regex) on line 200. (I didn't want to make functional changes)
2. move the `upperFirst` function on line 244 to before its first use
3. rename some unused function parameters from `prop` to `_prop`